tests/core/common/test_lbfgs_torch.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from itertools import product
+
+import numpy.testing as npt
+import pytest
+from ase.io import read
+from ase.optimize import LBFGS as LBFGS_ASE
+
+from fairchem.core.common.relaxation import OptimizableBatch
+from fairchem.core.common.relaxation.optimizers import LBFGS
+from fairchem.core.modules.evaluator import min_diff
+
+
+def test_lbfgs_relaxation(atoms_list, batch, calculator):
+    """Tests batch relaxation using fairchem LBFGS optimizer."""
+    obatch = OptimizableBatch(batch, trainer=calculator.trainer, numpy=False)
+
+    # optimize atoms one-by-one
+    for atoms in atoms_list:
+        atoms.calc = calculator
+        opt = LBFGS_ASE(atoms, damping=0.8, alpha=70.0)
+        opt.run(0.01, 20)
+
+    # optimize atoms in batch using ASE
+    batch_optimizer = LBFGS(obatch, damping=0.8, alpha=70.0)
+    batch_optimizer.run(0.01, 20)
+
+    # compare energy and atom positions, this needs pretty slack tols but that should be ok
+    for a1, a2 in zip(atoms_list, obatch.get_atoms_list()):
```

Comment:
I forget - are atoms optimized in place? i.e. what we're comparing here is indeed optimized states and not initial states.

Reply:
Yes, the positions/forces/stress in the batch are optimized in place